src/entity-manager/EntityManager.ts

```diff
@@ -1054,11 +1054,15 @@ export class EntityManager {
         where: FindOptionsWhere<Entity> | FindOptionsWhere<Entity>[] = {},
     ): Promise<number | null> {
         const metadata = this.connection.getMetadata(entityClass)
+        const column = metadata.columns.find(
+            (item) => item.propertyPath === columnName,
+        )
+
         const result = await this.createQueryBuilder(entityClass, metadata.name)
             .setFindOptions({ where })
             .select(
                 `${fnName}(${this.connection.driver.escape(
-                    String(columnName),
+                    column?.databaseName ?? String(columnName),
```

Comment:
I think it more makes sense to throw an error if column doesn't exist.

Reply:
Got it